dataflow/src/main/java/org/checkerframework/dataflow/cfg/ControlFlowGraph.java

```diff
@@ -303,4 +305,32 @@ public String toString() {
         String stringGraph = (String) res.get("stringGraph");
         return stringGraph == null ? super.toString() : stringGraph;
     }
+
+    /**
+     * Returns a string representation of this.
+     *
+     * @return a string representation of this
+     */
+    public String repr() {
```

Comment:
There already is a `toString` method in this class. Can you add cross-references and explain when to use which?